lib/charms/vault_k8s/v0/vault_tls.py

```diff
@@ -317,6 +317,20 @@ def get_tls_file_path_in_charm(self, file: File) -> str:
         storage_location = cert_storage.location
         return f"{storage_location}/{file.name.lower()}.pem"
 
+    def tls_file_available_in_charm(self, file: File) -> bool:
```

Comment:
How is the outcome of this function different to `tls_file_pushed_to_workload`?

Reply:
I answered this in the other question below.